spark/src/main/scala/org/apache/mahout/sparkbindings/SparkEngine.scala

```diff
@@ -263,15 +263,15 @@ object SparkEngine extends DistributedEngine {
       (implicit sc: DistributedContext):
     IndexedDatasetSpark = {
     val reader = new TextDelimitedIndexedDatasetReader(schema)(sc)
-    val ids = reader.readDRMFrom(src, existingRowIDs)
+    val ids = reader.readRowsFrom(src, existingRowIDs)
     ids
   }
 
   /**
-   * reads an IndexedDatasetSpark from default text delimited files
+   * Returns an [[org.apache.mahout.sparkbindings.indexeddataset.IndexedDatasetSpark]] from default text
```

Comment:
we probably need to choose between scala-style scaladoc multiline 

      /** Multiline Comment
       */

and Spark (java) -style multiline 

      /**
       * multiline comment
       */

I generally follow spark convention (i.e. scaladoc style for single line and the latter variation for multilines) but in this patch i see mix of both styles even in the same file.